Translations of `Errors` component updated

weblate opened the pull request:
Translations update from [Hosted Weblate](https://hosted.weblate.org) for [OSRD/Errors](https://hosted.weblate.org/projects/osrd/errors/).



Current translation status:

![Weblate translation status](https://hosted.weblate.org/widget/osrd/errors/horizontal-auto.svg)

gaetan-osrd commented on chart/templates/stateful_editoast/deployment.yaml:
```diff
@@ -13,7 +13,7 @@ metadata:
     {{- toYaml $mergedAnnotations | nindent 4 }}
 spec:
-  replicas: 1
+  replicas: {{ .Values.services.statefulEditoast.replicas | default 2 }}
```
I'm not against this change but it seems it should be in another PR

Khoyo replied:
It's already on dev :D

flomonster committed:
front: errors: translated using weblate (French)

Currently translated at 99.0% (203 of 205 strings)

Translation: OSRD/Errors
Translate-URL: https://hosted.weblate.org/projects/osrd/errors/fr/

Signed-off-by: Florian Amsallem <florian.amsallem@gmail.com>

Commit:
front: errors: add translation using weblate (Chinese (Simplified Han script))

Signed-off-by: Languages add-on <noreply-addon-languages@weblate.org>
